app/models/info_request.rb

```diff
@@ -1584,6 +1584,13 @@ def embargo_expiring?
     embargo ? embargo.expiring_soon? : false
   end
 
+  # Is the attached embargo still present but has reached its publication date
+  #
+  # Returns boolean
+  def embargo_pending_expiry?
```

Comment:
Its not the end of the world, but not sure what the differing method name adds here. `embargo_expired?` is a bit clearer in that its more obviously delegating to `embargo.expired?`.

Reply:
there's a different (and more accurate) `embargo_expired?` [over in pro#208](https://github.com/mysociety/alaveteli/pull/4296/files#diff-5f7dfe1185cac97cbc9f9dc356abce7cR1583) which is intended to only return true if the request has been published (and there was once an embargo), happy to come up with a better name here though - this describes an edge case which hopefully will only be true between midnight and the scheduled script run which deletes pending embargoes and causes the requests to be reindexed

Reply:
Ah right yeah, ISWYM.

I think this is fine in that case, and we can look at the `embargo_*` method names together when all merged to check each makes sense in context.

Reply:
is it worth me combining the 3 overlapping PRs into one once I've done the fixups so that we can review/check pre-merge?